src/org/loklak/harvester/TwitterScraper.java

```diff
@@ -332,26 +332,14 @@ private static String prepareSearchUrl(final String query, final ArrayList<Strin
                 continue;
             }
             // get images
-            if(images == null) images = new HashSet<>();
-            if ((p = input.indexOf("data-image-url=")) >= 0) {
-                String image_url = new prop(input, p, "data-image-url").value;
-                if (!image_url.endsWith(".jpg") && !image_url.endsWith(".png")) {
-                    DAO.log("strange image url: " + image_url);
-                }
-                images.add(image_url);
-                continue;
-            }
-            // get images
-            if ((p = input.lastIndexOf("background-image:url('")) >= 0) {
-                int q = input.lastIndexOf("'");
-                if (q > p + 22) {
-                    String image_url = input.substring(p + 22, q);
-                    if (!image_url.endsWith(".jpg") && !image_url.endsWith(".png")) {
-                        DAO.log("strange image url: " + image_url);
-                    }
-                    images.add(image_url);
+            String img_link;
```

Comment:
The scope can be reduced here.

Also, please provide some sample results (screenshot/link).

Reply:
uploaded the screenshot, please see

Reply:
@singhpratyush , I have fixed this issue, please see :)